Formula/r/radvd.rb

```diff
@@ -0,0 +1,68 @@
+class Radvd < Formula
+  desc "IPv6 Router Advertisement Daemon"
+  homepage "https://radvd.litech.org/"
+  url "https://github.com/radvd-project/radvd/releases/download/v2.20/radvd-2.20.tar.xz"
+  sha256 "25d2960fb977ac35c45a8d85b71db22ed8af325db7dbf4a562fb03eab2848dcd"
+  license "radvd"
+
+  head do
+    url "https://github.com/radvd-project/radvd.git", branch: "master"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "pkgconf" => :build
+  depends_on "libbsd"
+  uses_from_macos "bison" => :build
```

Comment:
Oh, I forgot about this. It is not a requirement but having a blank line between `depends_on` and `uses_from_macos` would improve readability a bit

```suggestion
  depends_on "libbsd"
  
  uses_from_macos "bison" => :build
```